src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthSimple.java

```diff
@@ -1,18 +1,10 @@
-////////////////////////////////////////////////////////////////////////////////
-// Test case file for checkstyle.
-// Created: Feb-2001
-// Ignore violation
-////////////////////////////////////////////////////////////////////////////////
 package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
 import java.io.*;
-/**
- * Contains simple mistakes:
- * - Long lines
- * - Tabs
- * - Format of variables and parameters
- * - Order of modifiers
- * @author Oliver Burn
- **/
+/*
+ * Config:
+ * max = 19
+ * countEmpty = true
```

Comment:
last comment: you can remove this line as `countEmpty  = true` by default and wasn't set explicitely inside test configuration

Reply:
That's right, I'll do it.

Reply:
Done.